Classes/Private/Manager/Database/OCTRealmManager.m

```diff
@@ -268,6 +301,17 @@ - (OCTMessageAbstract *)addMessageWithText:(NSString *)text
     return messageAbstract;
 }
 
+- (void)noteMessageFileChanged:(OCTMessageFile *)messageFile
+{
+    OCTMessageAbstract *abstractMessage = [messageFile linkingObjectsOfClass:[OCTMessageAbstract className] forProperty:@"messageFile"].firstObject;
```

Comment:
I think it would be better to do `forProperty:NSStringFromSelector(@selector(messageFile))` to get error if property will change.